docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -91,48 +77,47 @@ Make sure to also review our [app development best practices](/docs/integrations
+* You have shared your tax provider details with BigCommerce through email, and we've replied confirming the tax provider configuration is now ready on all supplied merchant and/or partner test stores (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
+* BigCommerce has shared the `provider_id` of the tax provider configuration with you through email (see [Sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce)).
+* You have built a draft app that successfully handles the OAuth flow during app installation (see [Building the app](#building-the-app)).
+* During the OAuth flow the app requests that the merchant authorize read and write permissions on the Information and Settings API scopes (see [Installing the app](#installing-the-app)).
+* During the OAuth flow the `client_id` and `access_token` was received and stored by you (see [Installing the app](#installing-the-app)).
 
-If the tax provider supports all eligible stores, then they may choose to provide an account registration flow in their app iFrame in order to capture these basic authentication credentials from merchants. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
+For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax provider connectivity. These basic credentials are used to authenticate each API request to the tax provider from the associated store, allowing you to reject any other unauthorized requests.
 
-If the tax provider is a private instance, then they may choose to provide the basic authentication credentials themselves.
+An account login flow allows you to capture basic authentication credentials from merchants. It's best to provide a login flow within your app iFrame so that you can choose your own authentication method. You can also provide an account registration flow in your iFrame to help merchants create a new tax provider account. Learn more about [designing the app UI](/docs/integrations/apps/guide/ui).
 
-In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after the tax provider's app has been successfully installed. Tax providers will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
+In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after your app has been successfully installed and the merchant's account credentials have been authenticated. You will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
```

Comment:
```suggestion
We recommend calling the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint immediately after the merchant installs your app and you authenticate the merchant's account credentials. Otherwise, your tax provider will not be displayed when merchants navigate to the **Settings > Setup > Tax** page in the control panel. You will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
```

Reply:
We can remove the next line (repetitive)

Reply:
Agreed, in the context of establishing a connection it's pretty low value.